letsencrypt/cli.py

```diff
@@ -315,7 +322,8 @@ def _handle_subset_cert_request(config, domains, cert):
              ", ".join(domains),
              br=os.linesep)
     if config.expand or config.renew_by_default or zope.component.getUtility(
-            interfaces.IDisplay).yesno(question, "Expand", "Cancel"):
+            interfaces.IDisplay).yesno(question, "Expand", "Cancel",
+                                       default=True):
```

Comment:
I think that minimising the number of lineages is almost always what the user wants, so I'm inclined to go with that as the default.  The alternatives are erroring or duplicating, I suppose, but I don't like the non-determinism that introduces.